src/applications/yellow-ribbon/actions/index.js

```diff
@@ -100,6 +101,17 @@ export const fetchResultsThunk = (options = {}) => async dispatch => {
       state,
     });
 
+    // Track the API request.
+    recordEvent({
+      event: 'edu-yellow-ribbon-search',
+      'edu-yellow-ribbon-name': name || undefined,
+      'edu-yellow-ribbon-state': state || undefined,
+      'edu-yellow-ribbon-city': city || undefined,
+      'edu-yellow-ribbon-contribution-amount': contributionAmount || undefined,
+      'edu-yellow-ribbon-number-of-students': numberOfStudents || undefined,
+      'edu-yellow-ribbon-number-of-search-results': response?.totalResults,
+    });
```

Comment:
Is `|| undefined` necessary vs. just using the variable? Or is the variable sometimes false-y but we need `undefined`?

Reply:
The latter! Per [this GitHub comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7199#issuecomment-620167496), they asked if falsey values could be `undefined` 🙂If these `|| undefined` statements weren't there, then it would have been `null` due to the code above this